Dockerfile

```diff
@@ -4,23 +4,16 @@ FROM rust:1.73-bookworm AS builder
 WORKDIR /usr/src/glaredb
 COPY . .
 
-RUN apt-get update && apt-get install -y openssl ca-certificates
 RUN cargo install just
 # Build release binary.
 RUN just build --release
 
-# Generate certs.
-RUN ./scripts/gen-certs.sh
-
 FROM debian:bookworm-slim
 
 # Runtime deps.
 RUN apt-get update -y && apt-get install -y openssl ca-certificates openssh-client
```

Comment:
I believe our tunnels feature relies on these.

Reply:
Yup, we'll want to keep these in.